firrtl/src/main/scala/firrtl/options/internal/WritableCircuitAnnotation.scala

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package firrtl.options.internal
+
+import firrtl.options.CustomFileEmission
+import firrtl.annotations.Annotation
+
+import java.io.File
+
+// Hack for enabling special emission of the ChiselCircuitAnnotation in WriteOutputAnnotations
+@deprecated("This trait is for internal use only. Do not use it.", "Chisel 5.0")
+trait WriteableCircuitAnnotation extends CustomFileEmission { self: Annotation =>
```

Comment:
We should soon move the `firrtl` package under `chisel3` to enable making this private. 🥲

Reply:
Agreed, good Chisel 6 change